src/serlio/utils/MELScriptBuilder.h

```diff
@@ -84,6 +84,7 @@ class MELScriptBuilder {
 	void setUndoState(bool undoState);
 
 	void getWorkspaceDir();
+	void getStringFromURL(const std::wstring& url);
 
 	void python(const std::wstring& pythonCmd);

```

Comment:
it looks like the `python` method is not used otherwise, consider avoiding calling python through MEL but use `MGlobal.executePythonCommand`

Reply:
doing this approach now. (however we need to call it twice, once for function (proc) declaration and then for retrieval, since [maya can only return python outputs for single line commands](https://discourse.techart.online/t/maya-calling-python-from-c-using-mglobal-executepythoncommand/5874).)